File: fibonacci.c
#include <stdio.h>
int fibonacci(int n);

// fn = fn-1 + fn-2
// f0 = 0
// f1 = 1


int fibonacci(int n)
{
    if (n <= 0)
        return 0;
    else if (n == 1)
        return 1;
    else
        return fibonacci(n - 1) + fibonacci(n - 2);
}

File: fibonacci.js
let number = prompt("enter index of fibonacci please :")

function fibonacci(x) {
    if (x == 0)
        return 0
    else if (x == 1)
        return 1
    else
        return fibonacci(x - 1) + fibonacci(x - 2)
}

console.log(`the ${number}th fibonacci number is ${fibonacci(number)}`)
